UP_onboarding_system/point_of_sale/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    MERCHANT = 1
+    CONSUMER = 2
+
+    ROLE_CHOICES = (
+        (MERCHANT, 'MERCHANT'),
+        (CONSUMER, 'CONSUMER')
+    )
+
+    name = models.CharField(max_length=50)
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    role = models.PositiveSmallIntegerField(choices=ROLE_CHOICES, default=2)
+
+    def __str__(self):
+        return self.name
+
+
+class Stores(models.Model):
+    name = models.CharField(max_length=50)
+    address = models.CharField(max_length=50)
+    lat = models.FloatField()
+    lng = models.FloatField()
+    merchant = models.ForeignKey(Profile, on_delete=models.CASCADE, related_name="stores")
+
+    def __str__(self):
+        return "{}: {}".format(self.name, self.address)
+
+    class Meta:
+        verbose_name_plural = 'Stores'
+
+
+class Items(models.Model):
+    name = models.CharField(max_length=50, unique=True)
```

Comment:
for name of item we have `unique=True` here, can u please explain why did u add it. 
We might have same items in different stores.

Reply:
According to me, As this is linked with Store , the unique property is of the items with 1 store . We can create same item in different store .